admin/controllers/pupil.js

```diff
@@ -1,6 +1,6 @@
 'use strict'
 
-const azureFileDataService = require('../services/data-access/azure-file.data.service')
+const azureBlobDataService = require('../services/data-access/azure-blob.data.service')
```

Comment:
Isn't this the equivalent of having the data service in the controller?

Reply:
yes, i'm not finished with the refactor yet.  this is still a draft PR.

step 1 - merge file and blob into 1 serviced
step 2 - eradicate superfluous methods from service
step 3 - refactor data services out